apps/nextjs-website/src/lib/types/webinar.ts

```diff
@@ -1,16 +1,28 @@
 import { Speaker } from '@/lib/types/speaker';
-import { Path } from './path';
+import { BlocksContent } from '@strapi/blocks-react-renderer';
 
 export type Webinar = {
   readonly description: string;
+  // Use textContent instead, remove when webinars will be managed by Strapi
+  // (remove it when both of this tasks
+  // https://pagopa.atlassian.net/browse/DEV-1557 and
+  // https://pagopa.atlassian.net/browse/DEV-1524 are resolved)
+  /** @deprecated */
   readonly html?: string;
+  readonly textContent?: BlocksContent;
```

Comment:
Maybe we should find a better name (such as `bodyContent`) for this attribute as this block can also support images and titles